index.html

```diff
@@ -244,14 +244,91 @@ <h2>
               </div>
             </li>
           </ul>
-          <ul class="info">
-            <a href="#">Frameworks</a>
-          </ul>
-          <ul class="info">
-            <a href="#">Skills</a>
-          </ul>
+          <section class="grids">
+            <div class="grid">
+              <img class="grid-image" src="./images/frameworks.png" alt="" />
+              <h3>Frameworks</h3>
+              <div class="txt1">
+                <div class="tags1">
+                  <li class="tag1">Bootstrap</li>
+                  <li class="tag1">Ruby on Rails</li>
+                </div>
+              </div>
+            </div>
+            <div class="grid">
+              <img class="grid-image" src="./images/skills.png" alt="" />
+              <h3>Skills</h3>
+              <div class="txt1">
+                <div class="tags1">
+                  <li class="tag1">Codekit</li>
+                  <li class="tag1">Github</li>
+                  <li class="tag1">Codepen</li>
+                  <li class="tag1">Gitlab</li>
+                </div>
+              </div>
+            </div>
+          </section>
```

Comment:
- [ ] Kindly remove this section as it is not part of the template you are working with. Since it is using green colors and the remaining part of the page has a purple theme going on it is not looking consistent with the rest of the design. I think it will not be pleasant from a user experience point of view. 🤝️